specification/dartLangSpec.tex

```diff
@@ -23420,25 +23425,14 @@ \subsubsection{Comments}
 \LMHash{}%
 \IndexCustom{Documentation comments}{documentation comments}
 are comments that begin with the tokens \code{///} or \code{/**}.
-Documentation comments are intended to be processed by
-a tool that produces human readable documentation.
-
-\LMHash{}%
-The current scope for a documentation comment immediately preceding
-the declaration of a class $C$ is the
-\IndexCustom{body scope}{scope!for statement body}
-of $C$.
-
-\LMHash{}%
-The current scope for a documentation comment immediately preceding
-the declaration of a non-redirecting generative constructor $k$
-with initializing formals is the formal parameter initializer scope of $k$
-(\ref{generativeConstructors}).
 
-\LMHash{}%
-Otherwise, the current scope for a documentation comment immediately preceding
-the declaration of a function $f$ is the formal parameter scope of $f$
-(\ref{formalParameters}).
+\rationale{%
+  Documentation comments are intended to be processed by
+  a tool that produces human readable documentation.
+  However, this document does not place any constraints
+  on the interpretation or use of documentation comments,
+  that is determined by each of the relevant tools.%
+}
```

Comment:
And this paragraph can be dropped too.

Saying nothing is better than saying something inconsequential or unnecessary.

Reply:
Removed.